script.js

```diff
@@ -13,3 +13,131 @@ close.addEventListener('click', () => {
+  },
+  {
+    dataId: 'dd6',
+    name: 'Website Portfolio',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    featuredImgBig: 'assets/image/firzz.png',
+    featuredImgSmall: 'assets/image/snapsmall.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://aneallaryea100.github.io/Portfolio/',
+    linkSource: 'https://github.com/aneallaryea100/Portfolio',
+    textModal: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  },
+];
+
+const cardSecz = document.querySelector('.cardSec');
+
+// loop data object
+for (let index = 0; index < data.length; index += 1) {
+  const dataIndex = data[index];
+  // container
+  const divContain = document.createElement('div');
+  divContain.setAttribute('class', 'card-works');
+  // head text
+  const nameHead = document.createElement('h5');
+  nameHead.setAttribute('class', 'ghg');
+  const nameText = document.createTextNode(dataIndex.name);
+  nameHead.appendChild(nameText);
+  divContain.appendChild(nameHead);
+  // description
+  const tagDescribe = document.createElement('p');
+  tagDescribe.setAttribute('class', 'ddestxts');
+  tagDescribe.setAttribute('id', 'farx');
+  const textDescribe = document.createTextNode(dataIndex.description);
+  tagDescribe.appendChild(textDescribe);
+  divContain.appendChild(textDescribe);
+  // technologies
+  const techList = document.createElement('ul');
+  techList.setAttribute('class', 'btn-hold');
+  const techEL = dataIndex.technologies;
+  techEL.forEach((el) => {
+    const elLi = document.createElement('li');
+    const elText = document.createTextNode(el);
+    elLi.appendChild(elText);
+    techList.appendChild(elLi);
+    divContain.appendChild(techList);
+  });
+  // button
+  const modalBtns = document.createElement('button');
+  modalBtns.setAttribute('class', 'esdd');
+  const textBtnx = document.createTextNode('See Project');
+  modalBtns.appendChild(textBtnx);
+  divContain.appendChild(modalBtns);
+  // add div to card section
+  cardSecz.appendChild(divContain);
+}
+
+/** ****** email validation ********* */
+const form = document.getElementById('form');
+const email = document.getElementById('mail');
+const errorMessage = document.querySelector('.error');
+
+form.addEventListener('submit', (element) => {
+  element.preventDefault();
+  const emailValue = email.value.trim();
+  const pat = /^[a-z]+$/;
+
+  if (pat.test(emailValue) === false) {
+    errorMessage.innerText = 'Email must be lowercase***';
+  } else {
+    form.submit();
+  }
```

Comment:
- [ ] The form is showing the error even when the email is lowercase, kindly fix this so your form work's as expected

![Screenshot_20220623_180804](https://user-images.githubusercontent.com/25227022/175355524-b905a325-f86e-456d-9bd0-977d55d17457.png)

Reply:
Form now accepts only lowercase emails.